app.js

```diff
@@ -63,6 +71,18 @@
       if ( this.$('.check.comment').is(':checked') ) {
         results = this.filterCommentResults(results);
       }
+      if ( this.$('.check.created').is(':checked') ) {
+        results = this.filterByCreatedDate(results);
+      }
+      if ( this.$('.check.updated').is(':checked') ) {
+        results = this.filterByUpdatedDate(results);
+      }
+
+      // Remove times from dates
+      _.each(results, function(macro) {
```

Comment:
I just snuck it into this PR, it doesn't really change anything except style.

I changed it for continuity (since I use underscore methods elsewhere now) and for style, since ZD apps best practices told me to: https://github.com/zendesk/demo_apps/blob/master/Bestpractices.md

I personally like the `.forEach` function better since it reads more like ruby, but I guess this isn't ruby :crying_cat_face: